x-pack/platform/plugins/shared/onechat/public/application/components/conversations/conversation_rounds/round_thinking/steps/tabular_data_result_step.tsx

```diff
@@ -40,34 +40,34 @@ export const TabularDataResultStep: React.FC<TabularDataResultStepProps> = ({
   return (
     <EuiFlexGroup direction="row" gutterSize="xs" alignItems="center">
       <EuiText size="s">
-        {i18n.translate(
-          'xpack.onechat.conversation.thinking.tabularDataResultStep.foundRecordsMessagePrefix',
-          {
-            defaultMessage: 'Found',
-          }
-        )}
+        <FormattedMessage
+          id="xpack.onechat.conversation.thinking.tabularDataResultStep.foundRecordsMessage"
+          defaultMessage="Found {results}"
+          values={{
+            results: (
+              <EuiLink
+                href={discoverUrl}
+                data-test-subj="onechat-esql-data-result-see-in-discover"
+                aria-label={i18n.translate(
+                  'xpack.onechat.conversation.thinking.tabularDataResultStep.seeInDiscoverAriaLabel',
+                  {
+                    defaultMessage: 'See results in Discover',
```

Comment:
```suggestion
                    defaultMessage: 'Explore results in Discover',
```

probably better for `a11y`?